constellation-app/constellation
# Enable renaming of Step Tabs in Data Access View

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change
Step Tabs in Data Access View can now be renamed by double clicking on them. This is also saved and loaded in the templates.


### Alternate Designs
n/a


### Why Should This Be In Core?
Better UX


### Benefits
Better UX


### Possible Drawbacks
If the user tries to open a new template with a previous version of Consty, com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException is thrown as explained in https://github.com/constellation-app/constellation/issues/1599


### Verification Process

1. Open DAV and rename the Step tab by double clicking on the default tab caption `Step 1` 
2. Add more tabs and rename 
3. Save template and see if it loads the new names correctly when the template is loaded again.



### Applicable Issues
https://github.com/constellation-app/constellation/issues/1599
